### PR TITLE
Feature/roadmap

### PR DESCRIPTION
**Pull request by nikitavoloboev**

### CODE REVIEW

File: docs/readme.mdx [hunk 1]
```diff
@@ -1,3 +1 @@
-# Learn Anything | Docs
```
Comment (johnletey) [inline docs/readme.mdx]: I think we should add this back ... I think the homepage looks a little to bland without any header IMO

File: docs/roadmap.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Roadmap
+
+## Project goal
```
Comment (johnletey) [inline docs/roadmap.mdx]:
```suggestion
## Goals
```

File: docs/readme.mdx (hunk 1 above)
Comment (nikitavoloboev) [reply docs/readme.mdx]: added back

File: docs/roadmap.mdx (hunk 2 above)
Comment (nikitavoloboev) [reply docs/roadmap.mdx]: done

**Comment by nikitavoloboev** [comment]
@johnletey check now, I added ideas doc

File: docs/readme.mdx (hunk 1 above)
Comment (johnletey) [reply docs/readme.mdx]: thanks!

File: docs/roadmap.mdx (hunk 2 above)
Comment (johnletey) [reply docs/roadmap.mdx]: wonderful!

File: docs/ideas.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- Google custom engine
+- YouTube
+- Arxiv
+
+## User actions
+
+- Explore
+- Learn
+- Add resources (edit/delete) --> only within a certain time-frame
+- Add topics (edit/delete) --> only within a certain time-frame
```
Comment (johnletey) [inline docs/ideas.mdx]: And also permissions for delete

File: docs/ideas.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- Bookmark resources
+- Follow topics
+- Add dependencies to topics
+- Create study plan
+
+## To consider
+
+- GDPR
```
Comment (johnletey) [inline docs/ideas.mdx]: Maybe we should clarify?

Comment (nikitavoloboev) [reply docs/ideas.mdx]: clarifd

File: docs/ideas.mdx (hunk 3 above)
Comment (nikitavoloboev) [reply docs/ideas.mdx]: what do you mean?

Comment (johnletey) [reply docs/ideas.mdx]: Well ... not any user can just go and delete a resource/topic!

**Pull request merged**